File: 2024/august/java/01.java
class Solution {
    
    private int solve(String s){
        int age = 0;
        for (int i = 11; i <= 12; ++i){
            age = age * 10 + (s.charAt(i)-'0');
        }
        return age > 60 ? 1 : 0;
    }
    public int countSeniors(String[] details) {
        int ans = 0;
        for (var cur : details){
            ans+= solve(cur);
        }
        return ans;
    }
}

File: 2024/august/javaScript/01.js
/**
 * @param {string[]} details
 * @return {number}
 */
var countSeniors = function(details) {
    const solve = function(s){
        let age = 0;
        for (let i = 11; i <= 12; ++i){
            age = age * 10 + (s[i]-'0');
        }
        return age > 60 ? 1 : 0;
    };
   
        let ans = 0;
        for (const cur of details){
            ans+= solve(cur);
        }
        return ans;
};
